autonomi/src/client/high_level/data/helpers.rs

```diff
@@ -244,13 +260,15 @@ impl Client {
         {
             Ok((receipt, free_chunks)) => (receipt, free_chunks),
             Err(err) => {
```

Comment:
Tiny one, but if possible, it would be great to add some logging here for the failure case.